frontend/src/components/board/SidebarItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { MenuItem } from "../../interfaces/canva-interfaces";
+
+interface RenderItemProps {
+  item: MenuItem;
+  activeItem: string | null;
+  color: string;
+  size: number;
+  handleColorChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleSizeChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleClick: (name: string, action?: () => void) => void;
+}
+
+const RenderItem: React.FC<RenderItemProps> = ({
```

Comment:
Maybe other name would be more suitable?
```suggestion
const SidebarItem: React.FC<SidebarItemProps> = ({
```